docs/source/quick_start.rst

```diff
@@ -0,0 +1,193 @@
+============
+Quick Start
+============
+
+  This guide is intended to get new Kokkos users started in less than one hour.  Kokkos Core is the foundation for all other Kokkos Ecosystem projects.
+
+------------
+Requirements
+------------
+
+::
+
+  CMake >=3.18
+  gcc >= 8.3.1
+  CUDA >= 11
+  rocm >=5.7
+
+--------------------------------------
+Download Current Release, Set Up Build 
+--------------------------------------
+
+..
+  Choice 1 for obtaining current release:
+..
+
+* `Click to Download Current Release <https://github.com/kokkos/kokkos/archive/refs/heads/master.zip>`_ 
+
+
+..
+  Temp. Notes --
+  Buttons, badges, icons:
+  https://sphinx-design.readthedocs.io/en/latest/badges_buttons.html
+  :bdg-link-primary-line:`explicit title <https://example.com>`
+
+  https://getbootstrap.com/docs/5.0/components/badge/
+..
+
+
+..
+  Choice 2 for obtaining current release:
+..
+
+:bdg-link-primary:`Current Release <https://github.com/kokkos/kokkos/archive/refs/heads/master.zip>`
+
+
+::
+
+  unzip kokkos-master.zip
+  cd kokkos-master
+  mkdir build
+  cd build
```

Comment:
This isn't being syntax highlighted for me.

Also if you change the URL as I suggested you will need to change this:

```suggestion
.. code-block:: sh

  tar -xf kokkos-4.2.01.tar.gz
```
(Note the second URL I suggested gives a weird tarball name idk why)